lib/MatterbridgeManager.php

```diff
@@ -383,6 +383,7 @@ private function generateConfig(array $bridge): string {
 					$serverUrl = $part['server'];
 				} else {
 					$serverUrl = preg_replace('/\/+$/', '', $this->urlGenerator->getAbsoluteURL(''));
+					$content .= "	SeparateDisplayName = true" ."\n";
```

Comment:
Does this mean all configs have to be recreated?

Reply:
Yes. Until the config is recreated, it will continue to function as before.